flake.nix

```diff
@@ -4,8 +4,8 @@
   inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs/nixpkgs-unstable";
   inputs.fenix.url = "github:nix-community/fenix";
   inputs.fenix.inputs.nixpkgs.follows = "nixpkgs";
-  inputs.nixd.url = "github:nix-community/nixd";
-  inputs.nixd.inputs.nixpkgs.follows = "nixpkgs";
+  inputs.nil.url = "github:oxalica/nil";
+  inputs.nil.inputs.nixpkgs.follows = "nixpkgs";
```

Comment:
Do we need to use the github nil, can we use the one in nixpkgs?

Reply:
The one in nixpkgs is around a year old [nil 2023-08-09](https://github.com/oxalica/nil/releases/tag/2023-08-09). But it shouldn't cause any issues using the old version (I tested both already)